src/java.base/share/classes/java/lang/reflect/code/op/CoreOps.java

```diff
@@ -1857,6 +1857,7 @@ public static VarOp create(OpDefinition def) {
             String name = def.extractAttributeValue(ATTRIBUTE_NAME, true,
                     v -> switch (v) {
                         case String s -> s;
+                        case null -> null;
```

Comment:
What motivated this change?

Reply:
NPE when the attribute is missing. This passes the null name and works OK.

Reply:
Ah, yes (also i think the change from `Map.of` to `new HashMap` is related to that). I think that's originating from building the code model to quote the lambda. Do you mind if i fix this separately? Lets add variable names to that code model (derived from the parameter index) to avoid this issue for now?

Reply:
OK, I've added parameter names and reverted the fix.